Tools/machines/lumi-csc/submit.sh

```diff
@@ -27,4 +25,19 @@ export FI_MR_CACHE_MONITOR=memhooks  # alternative cache monitor
 export ROCFFT_RTC_CACHE_PATH=/dev/null
 
 export OMP_NUM_THREADS=1
-srun ../warpx inputs > outputs
+
+cat << EOF > select_gpu
+#!/bin/bash
+
+export ROCR_VISIBLE_DEVICES=\$SLURM_LOCALID
+exec \$*
+EOF
+
+chmod +x ./select_gpu
+
+CPU_BIND="map_cpu:48,56,16,24,1,8,32,40"
+
+export MPICH_GPU_SUPPORT_ENABLED=1
+
+srun --cpu-bind=${CPU_BIND} ./select_gpu ./warpx inputs
```

Comment:
We usually do this to have a common output file that we can programmatically parse later.
```suggestion
srun --cpu-bind=${CPU_BIND} ./select_gpu ./warpx inputs | tee outputs.txt
```

Reply:
sure... you are right